experimental/dds/tree2/src/feature-libraries/typed-schema/typedTreeSchema.ts

```diff
@@ -319,6 +319,7 @@ export class FieldSchema<Kind extends FieldKind = FieldKind, Types = AllowedType
 	 */
 	public constructor(public readonly kind: Kind, public readonly allowedTypes: Types) {}
 
+	// TODO: cache the result of this getter
```

Comment:
TODO for later? Is there a work item for it?

Reply:
ADO item exists, will add reference to it